dev/tools/controllerbuilder/pkg/annotations/fileannotation.go

```diff
@@ -34,7 +35,17 @@ func (a *FileAnnotation) FormatGo() string {
 	var sb strings.Builder
 
 	fmt.Fprintf(&sb, "// %s\n", a.Key)
-	for k, values := range a.Attributes {
+
+	// Get all keys and sort them for consistent ordering
+	keys := make([]string, 0, len(a.Attributes))
```

Comment:
This isn't required, but having a consistent ordering might make it easier for us to detect file diff in the generated files in the future.